src/platform/forms-system/src/js/widgets/CheckboxWidget.jsx

```diff
@@ -15,6 +15,13 @@ export default function CheckboxWidget({
     <span className="form-required-span">(*Required)</span>
   ) : null;
   const widgetClasses = classNames('form-checkbox', options.widgetClassNames);
+
+  const { widgetProps } = options;
+
+  const getProps = val => ({
+    ...(widgetProps?.[val] || {}),
```

Comment:
I recommend simply `widgetProps?.[val] || {}` over `({ ...(widgetProps?.[val] || {} })`. No need to spread an object into an object literal when it's the only thing in the literal. :smile: